src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx

```diff
@@ -88,6 +88,10 @@ export function GroupContainer({
     if (container.edit?.multiPage) {
       setMultiPageIndex(0);
     }
+
+    if (container.edit?.openByDefault === true && repeatingGroupIndex === -1) {
```

Comment:
` === true` is redundant here.
Also you can start this effect with
```js
const {edit} = container;
if (!edit) {
  return;
}
```

Reply:
Done and done 💯